dclab/definitions/meta_const.py

```diff
@@ -124,6 +124,37 @@
         ["target duration", float, "Target measurement duration [min]"],
         ["target event count", fint, "Target event count for online gating"],
     ],
+    # All qpi-related keywords
+    "qpi": [
+        # experiment-related qpi metadata, see qpretrieve for details
+        ["wavelength", float, "Laser wavelength [nm]"],
+        ["medium index", float, "Refractive index of Medium"],
+        # post-analysis-related qpi metadata
+        ["software version", str, "Software version used"],
+        # calculation of pha and amp from hologram
+        ["padding", fbool, "If padding was used"],
+        ["subtract_mean", fbool, "Subtract mean of hologram before FFT"],
+        # pipeline_kws
+        ["filter_name", str, "Fourier filter used"],
+        ["filter_size", str, "Fourier filter size [1/pix]"],
+        ["filter_size_interpretation", str, "How to interpret filter size"],
+        ["scale_to_filter", fboolorfloat, "Crop Fourier image before iFFT"],
+        ["sideband_freq", f1dfloattuple,
+         "Frequency coordinates of the sideband [1/pix]"],
+        ["invert_phase", fbool, "Invert the phase data"],
+        # "pixel_size_qpi" depends on `scale_to_filter`.
+        # If `scale_to_filter` is False, this does not change.
+        # If `scale_to_filter` is not False, this value will differ from
+        # `qpi_holo` `pixel size`.
+        ["pixel_size_qpi", float, "Image space pixel size after iFFT [um]."],
```

Comment:
pixel_size_oah -> original hologram

pixel_size_qpi -> of pha/amp due to possible FT cropping.

Reply:
And pixel_size_qpi is the same as normal rtdc pixel size